content/md/posts/clojure-which-execution-option-to-use.md

```diff
@@ -0,0 +1,217 @@
+{:title "Clojure CLI tools - which execution option to use"
+ :layout :post
+ :date "2021-12-20"
+ :topic "clojure-cli"
+ :tags  ["clojure-cli" "tools-deps"]}
+
+Clojure CLI tools provide a very flexible way to run Clojure, using aliases to include community libraries and tools to enhance clojure projects and provide independent tools. Understand the execution options (exec-opts) on the command line options ensures an effective use of Clojure CLI tools.
+
+Clojure CLI tools was [first released in February 2020](https://clojure.org/releases/tools#v1.10.1.510) with a specific focus on running Clojure code quickly and easily, which also required the management of dependencies from Maven, Git and a local file space.
```

Comment:
Although that's all the releases page lists, the CLI was actually first released in December 2017 and we switched to it in 2018 and were using it heavily back then -- so it's over two years older than you state here!

Reply:
I didnt realize it was that old.  Changed text to first documented release :)